product_extended_segmentation/model/installer.py

```diff
@@ -1,35 +1,28 @@
 # -*- coding: utf-8 -*-
-from openerp.osv import fields, osv
+from openerp.osv import osv
+from openerp import api, fields
 
 
 class ProductExtendedSegmentationSettings(osv.osv_memory):
     _inherit = 'purchase.config.settings'
-    _columns = {
-        'std_price_neg_threshold': fields.float(
-            string="Standard Price Negative Threshold (%)",
-            help=('Maximum percentage threshold that Standard Price is '
-                  'allowed to lower')),
-    }
+    std_price_neg_threshold = fields.Float(
+        string="Standard Price Bottom Threshold (%)",
+        help=('Maximum percentage threshold that Standard Price is '
+              'allowed to lower')
```

Comment:
the same help and deafult than on company to be conssitent bth fields must be defined cpying and pasting.